sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/rel/BeamAggregationRel.java

```diff
@@ -80,7 +80,8 @@ public BeamAggregationRel(RelOptCluster cluster, RelTraitSet traits
         BeamSqlRelUtils.getBeamRelInput(input).buildBeamPipeline(inputPCollections, sqlEnv);
     if (windowFieldIdx != -1) {
       upstream = upstream.apply(stageName + "assignEventTimestamp", WithTimestamps
-          .of(new BeamAggregationTransforms.WindowTimestampFn(windowFieldIdx)))
+          .of(new BeamAggregationTransforms.WindowTimestampFn(windowFieldIdx))
+          .withAllowedTimestampSkew(new Duration(Long.MAX_VALUE)))
```

Comment:
Two comments here:

* Will other DoFns we defined in BeamSQL have the similar issue? (The classes in `org.apache.beam.sdk.extensions.sql.impl.transform` package)
* In the future, the allowed timestamp skew might be better passed in as a config to the BeamSql api?(I am ok to keep it hardcoded in this PR.)

Reply:
1. This is the only place where we assign EventTimestamp explicitly;
2. We may not need this parameter, now the optional aligned time in `tumble`, `hop` and `session` is used as max_deplay_time;